clear is_cloned flag on translation request (WP-927) (#554)
Smartling/wordpress-localization-plugin: readme.txt

3. Track translation status within WordPress from the Submissions Board. View overall progress of submitted translation requests as well as resend updated content.

== Changelog ==
= 3.10.0 =

> * Changed submission cloned status, now set to false when translating a previously cloned submission

Reviewer: Reworded. "set to false" sounds too technical.
```suggestion
* Remove a cloned status when the user requests translation of the previously cloned submission
```
